cmd/podman/common.go

```diff
@@ -352,6 +352,10 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 		"network", getDefaultNetwork(),
 		"Connect a container to a network",
 	)
+	createFlags.Bool(
+		"no-cgroups", false,
```

Comment:
`--cgroup-manager=none` makes better sense?

In rootless docker, `docker info` shows  `Cgroup Driver: none`.

Reply:
Maybe we can find a flag-name starting with `cgroup`? It would be nice to have them grouped together in the man page and in the help messages.

Can't we replicate `--cgroup-manager` here as well? Cobra should support that and allow for over-writing the value in sub-commands.